### DISCUSSION TITLE
PostRector rule

### DISCUSSION THREAD

**Discussion by lugosium** [rectorphp/rector#8176] [Q&A] [answered]
Hello,

After upgrade to 0.18 i got an error:

```
 [ERROR] Expected an instance of this class or to this class among its parents
         "Rector\Core\Contract\Rector\RectorInterface". Got: "Rector\PostRector\Rector\NameImportingPostRector"
```

for this line:

`$rectorConfig->rule(NameImportingPostRector::class);`

Do anyone know why? Is this an invalid rule import?

Thank you

**Comment by samsonasik** [comment, marked as answer]
PostRector is automatically called, you don't need to call it.

**Reply by lugosium** [reply]
Thank you 👍
